20180908/jobCentTelegramBot/sponsor/server/controllers/index.js

```diff
@@ -1,5 +1,11 @@
+<<<<<<< HEAD
```

Comment:
Looks like a merge conflict you forgot to remove

Reply:
+1